modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -674,12 +674,25 @@ definition {
 	}
 
 	@description = "LPS-129928 - Verify that the Publish button turns unavailable after the Workflow is published"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test PublishButtonTurnsUnavailableAfterPublish {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#PublishButtonTurnsUnavailableAfterPublish";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start Task Connector",
+			connectorOut = "start",
+			key_direction = "top");
+
+		PortletEntry.publish();
 
-		// TODO LRQA-73001 PublishButtonTurnsUnavailableAfterPublish pending implementation
+		Workflow.viewDefinitionPublished(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.viewButtonPublishNotPresent();
```

Comment:
Incorrect flow.

After publishing, the page is now maintained, so the macro created (viewButtonPublishNotPresent), must be called right after this action.

Reply:
Done